app/controllers/v1/supported_ssgs_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+module V1
+  # API for Supported SSGs mapped to RHEL minor versions
+  class SupportedSsgsController < ApplicationController
+    def index
+      render_json scope_search
```

Comment:
It's a bit weird to use `scope_search` here since `SupportedSsg` isn't backed by the DB, but I think we may be able to still define scoped searches against it, potentially filtering by OS minor version or something in the future, but I've never tried to use it with a model that isn't backed by ActiveRecord.

Reply:
There is definitely room for improvement. There wasn't a way to provide an API endpoint without a db resource. 

The `scope_search` definition is/was needed for dependent methods. I can check if it is still required, as there is `metadata` override.

Reply:
Removed the use of `scope_search`.